input/images/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv

```diff
@@ -3299,13 +3299,13 @@ Decision: Births should behave the same as deaths.",Needs work,FHIR-41387,Extens
 3,Mortality,NA,NA,NA,Death Record Identifier,*NO IJE MAPPING*,"YYYYJJNNNNNN,  YYYY = death year JJ = jurisdiction  and NNNNNN = certificate number",,VRDR,DeathCertificateDocument,identifier.value,string(12),"YYYYJJNNNNNN,  YYYY = death year JJ = jurisdiction  and NNNNNN = certificate number",,,,,,DeathCertificateDocument,
 4,Mortality,NA,NA,NA,Death Record Identifier,*NO IJE MAPPING*,"YYYYJJNNNNNN,  YYYY = death year JJ = jurisdiction  and NNNNNN = certificate number",,VRDR,CauseOfDeathCodedContentBundle,identifier.value,string(12),"YYYYJJNNNNNN,  YYYY = death year JJ = jurisdiction  and NNNNNN = certificate number",,,,,,CauseOfDeathCodedContentBundle,
 5,Mortality,NA,NA,NA,Death Record Identifier,*NO IJE MAPPING*,"YYYYJJNNNNNN,  YYYY = death year JJ = jurisdiction  and NNNNNN = certificate number",,VRDR,DemographicCodedContentBundle,identifier.value,string(12),"YYYYJJNNNNNN,  YYYY = death year JJ = jurisdiction  and NNNNNN = certificate number",,,,,,DemographicCodedContentBundle,
-,Natality,NA,NA,NA,Record Identifier,*NO IJE MAPPING*,"YYYYJJNNNNNN,  YYYY = year JJ = jurisdiction  and NNNNNN = certificate number",,BFDR,BundleDocumentBirthReport,identifier.value,string(12),"YYYYJJNNNNNN,  YYYY = year, JJ = jurisdiction,  and NNNNNN = certificate number",,,,,,BundleDocumentBirthReport,
```

Comment:
I saw this in the BFDR PR as well, but curious as to why this is getting deleted?